scripts/hf_sim.py

```diff
@@ -528,16 +540,18 @@ def validate_end(idx_n):
     t0 = MPI.Wtime()
     in_stats = mkstemp()[1]
 
-    vm = args.hf_vel_mod_1d
+    v1d_path = args.hf_vel_mod_1d
     for s in range(work.size):
-        if args.site_vm_dir != None:
-            vm = os.path.join(args.site_vm_dir, "%s.1d" % (stations_todo[s]["name"]))
+        if args.site_v1d_dir is not None:
+            v1d_path = os.path.join(
+                args.site_v1d_dir, f"{stations_todo[s]['name'].decode('ascii')}.1d"
```

Comment:
I'd prefer if this was called with `str(stations_todo[s]['name'])` Thoughts @jpa198 

I've just been bitten by bytestrings too much before to be comfortable with changes to/from decode.